Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -845,10 +852,18 @@
     fireModes:
     - proto: BulletLaserSpread
       fireCost: 150
+      color: "#5fcde4"
     - proto: BulletLaserSpreadNarrow
       fireCost: 200
+      color: "#5fcde4"
     - proto: BulletDisablerSmgSpread
       fireCost: 120
+      color: "#ff0006"
+    clothingVisuals:
+      back:
+      - state: equipped-BACKPACK-color
+      suitstorage:
+      - state: equipped-SUITSTORAGE-color
```

Comment:
Add inhands! Makes it readable for other players at a glance and it looks like the sprites have room for it. They're found in `Battery\inhands_64x.rsi`.